ATACseq_footprinting/sort.sh

```diff
@@ -81,3 +81,30 @@ else
       echo "sorted merged bams already indexed"
 fi      
 
+###########################---STEP 6: PEAK CALLING WITH MACS2---########################################
+if [ ! -f "$bam_path/peak_calling/${PREFIX}/${PREFIX}_raw.bed" ]; then
+
+      if [ ! -f "$bam_path/peak_calling" ]; then
+            mkdir "$bam_path/peak_calling/"
+      fi
+      if [ ! -f "$bam_path/peak_calling/${PREFIX}" ]; then
+            mkdir "$bam_path/peak_calling/${PREFIX}"
+      fi
+      
+      module load macs2 
+      gsize=2620345972 #is this correct?!
+      echo "Running macs2 with .bam file: ${PREFIX}.merged.sorted.bam"
+
+      #Arguments:
+      #-t: the IP data file
+      #--outdir: the output directory
+
+      macs2 callpeak -t "$bam_path/${PREFIX}.merged.sorted.bam" --name ${PREFIX} --outdir "$bam_path/peak_calling/${PREFIX}" --gsize $gsize --nomodel --shift -100 --extsize 200 --broad 
+      cp "$bam_path/peak_calling/${PREFIX}/{PREFIX}_peaks.broadPeak" "$bam_path/peak_calling/${PREFIX}/{PREFIX}_raw.bed"
```

Comment:
The parameters/settings for MACS2 command should be variables to enter into the pipeline script. 
Specifically this part:
--gsize $gsize --nomodel --shift -100 --extsize 200 --broad

I think $gsize is already a variable, but maybe it would be good to make this part more customizable and have it use variables defined at the beginning of the script or as inputs to the script?